src/tests/ftest/recovery/pool_cleanup.py

```diff
@@ -34,9 +34,9 @@ def test_corrupt_label_ms(self):
 
         Jira ID: DAOS-11741
 
-        :avocado: tags=all,pr
-        :avocado: tags=vm
-        :avocado: tags=recovery,pool_cleanup
+        :avocado: tags=all,daily_regression
+        :avocado: tags=hw,medium
```

Comment:
And here
https://github.com/daos-stack/daos/blob/1d3929216fe5d1cbae325887c0e2487fec5ed514/src/tests/ftest/recovery/pool_cleanup.yaml#L13-L16